mediapipe/tasks/python/vision/image_segmenter.py

```diff
@@ -64,27 +71,22 @@ class ImageSegmenterOptions:
       objects on single image inputs. 2) The video mode for segmenting objects
       on the decoded frames of a video. 3) The live stream mode for segmenting
       objects on a live stream of input data, such as from camera.
-    output_type: The output mask type allows specifying the type of
-      post-processing to perform on the raw model results.
-    activation: Activation function to apply to input tensor.
+    output_confidence_masks: Whether to output confidence masks.
+    output_category_mask: Whether to output category mask.
     result_callback: The user-defined result callback for processing live stream
       data. The result callback should only be specified when the running mode
       is set to the live stream mode.
   """
 
-  class OutputType(enum.Enum):
-    UNSPECIFIED = 0
-    CATEGORY_MASK = 1
-    CONFIDENCE_MASK = 2
-
   class Activation(enum.Enum):
```

Comment:
Activation setting is deprecated as well.

Reply:
@yichunk I've removed it now.